components/admin-cards/UpdateCardModal.tsx

```diff
@@ -438,6 +451,21 @@ export default function UpdateCardModal({
                 </>
               )}
             </Stack>
+            {card.image_url && (
+              <Stack className="flex justify-center items-center">
+                <Image
+                  className={`cursor-pointer`}
+                  src={`https://simulationhockey.com/tradingcards/${card.image_url}`}
+                  fallback={
+                    <div className="relative z-10">
+                      <Image src="/cardback.png" />
+                      <div className="absolute top-0 left-0 w-full h-full bg-black opacity-50 z-20"></div>
+                    </div>
+                  }
+                  alt={`${card.player_name} Card`}
+                />
```

Comment:
could we use https://github.com/GuthrieW/shl-trading-cards/blob/main/components/images/TradingCard.tsx

instead?